components/uploadFormHook.tsx

```diff
@@ -89,20 +88,9 @@ export const UploadFormHook: FC = () => {
             </p>
           </div>
 
-          <label className="mb-5" htmlFor="update">
-            写真を撮影してアップロードする
-          </label>
-          <br />
-          <input
-            className="mb-5"
-            id="upload"
-            type="file"
-            accept="image/*"
-            capture="environment"
-            {...register('photo')}
-            required={true}
-          />
-          <label />
+          <div>
+            <SendImageForm />
+          </div>
         </div>
         <button
```

Comment:
```suggestion
        </div>
        <br />
        <button
```
IMO the buttons would be separated with some space.

Reply:
I commited this suggestion to the code.